traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/BillingInformationCollectionFragment.java

```diff
@@ -179,11 +176,21 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
             holder.label.setText(a.getLabel());
             holder.value.setText(a.getValue());
             holder.delete.setTag(position);
+            holder.itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    selectedIndex = position;
+
+                    adapter.notifyDataSetChanged();
```

Comment:
notifyItemChanged(pos) should work here, would make insertion smoother

hmm, not sure why tagging (`holder.delete.setTag(position);`) is used. we should be able to use the position param of `onBindViewHolder(@NonNull ViewHolder holder, int position) {`